front/src/modules/conflict/components/ConflictCardV2.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+import { useTranslation } from 'react-i18next';
+import { BsLightningFill } from 'react-icons/bs';
+
+import { msToIsoDate } from 'utils/date';
+
+import type { ConflictWithTrainNames } from '../types';
+
+export default function ConflictCardV2({
+  conflict,
+  onConflictClick,
+}: {
+  conflict: ConflictWithTrainNames;
+  onConflictClick: (conflict: ConflictWithTrainNames) => void;
+}) {
```

Comment:
And export default at bottom
```suggestion
const ConflictCardV2 = ({
  conflict,
  onConflictClick,
}: {
  conflict: ConflictWithTrainNames;
  onConflictClick: (conflict: ConflictWithTrainNames) => void;
}) => {
```

Reply:
Done